### PR TITLE
feat: allow API for counting contributing developers

### PR DESCRIPTION
**Pull request by Kirill89**
- [ ] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/broker/blob/master/.github/CONTRIBUTING.md) rules

#### What does this PR do?

Allow API for counting contributing developers.

#### Additional questions

https://snyksec.atlassian.net/browse/COMET-315

### CODE REVIEW

File: client-templates/github-enterprise/accept.json.sample [hunk 1]
```diff
@@ -641,7 +641,7 @@
     {
       "//": "used to get repo's contributors list",
       "method": "GET",
-      "path": "/repos/:owner/:repo/contributors",
+      "path": "/repos/:owner/:repo/stats/contributors",
```
Comment (yuliabaron) [inline client-templates/github-enterprise/accept.json.sample]: are you sure regarding `stats`?
are we using this API https://developer.github.com/v3/repos/#list-contributors ?

Comment (Kirill89) [reply client-templates/github-enterprise/accept.json.sample]: https://developer.github.com/v3/repos/statistics/#get-contributors-list-with-additions-deletions-and-commit-counts

Comment (Kirill89) [reply client-templates/github-enterprise/accept.json.sample]: Yes, you right!

Comment (Kirill89) [reply client-templates/github-enterprise/accept.json.sample]: 👁 🦅

**Pull request merged**